kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -283,7 +283,7 @@ namespace Kratos
         // Initialize cut edges vectors and points arrays
         unsigned int n_cut_edges = 0;
         rIntersectionPointsArray.clear();
-        array_1d<unsigned int, mNumEdges> cut_edges_vector = ZeroVector(mNumEdges);
+        array_1d<unsigned int, mNumEdges> cut_edges_vector(mNumEdges, 0);
```

Comment:
ouch, what a nasty conversion problem
Interesting this didn't show up before

did you have this problem in the Altair CI?

https://github.com/KratosMultiphysics/Kratos/blob/284902a7f196d4c19df9fda53d7bf273f9914016/kratos/includes/ublas_interface.h#L80

Reply:
Actually no, in Altair ci we are only compiling centos in Release, where the check is not being done.